src/shoobx/mocks3/tests/test_config.py

```diff
@@ -5,11 +5,11 @@
 ###############################################################################
 """Shoobx S3 Config Test
 """
-import mock
 import os
 import shutil
 import tempfile
 import unittest
+from unittest import mock
```

Comment:
can't see mock usage added in the diff, pyflakes does not complain?

Reply:
this is pyupgrade changing from `import mock` to `from unittest import mock`